skymarshal/dexserver/dexserver.go

```diff
@@ -15,14 +17,15 @@ import (
 )
 
 type DexConfig struct {
-	Logger       lager.Logger
-	IssuerURL    string
-	WebHostURL   string
-	ClientID     string
```

Comment:
was this ClientID and ClientSecret was used by skymarshal to talk to Dex ?

Reply:
Yup we used to only need to configure one client for all of concourse, but now since we need to configure multiple clients (web, tsa). We just pass in a list of clients: https://github.com/concourse/concourse/pull/4950/files/cd34c1ccbeb2bc773aedf7bef60629a6083c7602..02fe4b927dc8cc3290842fac77e61c6bee3bb588#diff-097bd19334894da24754e934c93e4638R25